src/themes/greyVest/ErrorList.js

```diff
@@ -1,16 +1,12 @@
 import React from 'react'
-import _ from 'lodash/fp'
-import ErrorBlock from './ErrorBlock'
 import ErrorText from './ErrorText'
 
-let ErrorList = ({ block = false, children }) =>
-  _.map(
-    e =>
-      block ? (
-        <ErrorBlock key={e}>{e}</ErrorBlock>
-      ) : (
-        <ErrorText key={e}>{e}</ErrorText>
-      ),
-    _.castArray(children)
+let ErrorList = ({ Component = ErrorText, children, ...props }) =>
```

Comment:
At this point, this isn't an `ErrorList`,  it's basically just a `List` that happens to default to `ErrorText`. Something this flexible starts to become of questionable value. The benefit of this:

```jsx
<List Component={SomeComponent} someProp="value">
  {['wrap', 'these']}
</List>
```
over this:

```jsx
_.map(
  x => <SomeComponent someProp="value">{x}<SomeComponent>
  ['wrap', 'these']
)
```

isn't super clear, except that it's a thing you need to know now and gets pretty close to a "foreach" component (which is generally discouraged by the react community as it starts to get into a template language kinda thing instead of just being plain js). I suppose the ideal case for it might be wrapping actual JSX components:

```jsx
<List Component={SomeComponent} putThisProp="onAllChildren">
  <Wrap />
  <These />
  <Components />
</List>
```

which I suppose is better than

```js
_.map(
  x => <SomeComponent putThisProp="onAllChildren">{x}<SomeComponent>
  [
    <Wrap />,
    <These />,
    <Components />
  ]
)

```

Still, it definitely seems like a pure preference thing that doesn't have a clear answer. This seems like something we should open up for discussion by the rest of the team 🤔. I don't feel strongly about it, but it does strike as me something we might not want to do. If we do proceed, we should definitely factor out List and make ErrorList just be `defaultProps` on List.

Reply:
I was about to say that we don't have anything that would justify a generic `List` component, though actually, we kinda do -- there's currently no list markup anywhere in contexture-react, but one could argue that there *should* be, for semantic reasons, and a `List` that wraps its children in `<li>` would accomplish that pretty well.

(I doubt anyone will argue that it's worth our time to pursue semantic HTML standards right this second, but since it might one day be necessary for accessibility stuff, might as well bring it up where it's relevant.)

In defense of `ErrorList`, it seems to me that the intent, based on the majority of its actual usage (eg. autoform fields), was less "list of errors" and more "an error (but possibly more than one)". In practice, it almost always holds only one error -- could be something to keep in mind if we do decide to use `List`.